server/src/main/java/io/seata/server/console/impl/db/GlobalLockDBServiceImpl.java

```diff
@@ -55,13 +55,14 @@
     private String dbDataSource;
 
     @Override
-    public PageResult<GlobalLockVO> queryByTable(String tableName) {
+    public PageResult<GlobalLockVO> query(GlobalLockParam param) {
         List<GlobalLockVO> list = new ArrayList<>();
         Connection conn = null;
         PreparedStatement ps = null;
         ResultSet resultSet = null;
         DataSource dataSource = EnhancedServiceLoader.load(DataSourceProvider.class, dbDataSource).provide();
-        String queryAllLockSQL = LockStoreSqlFactory.getLogStoreSql(dbType).getAllLockSQL(lockTable, tableName);
+        //need to check the param to assemble the sql
+        String queryAllLockSQL = LockStoreSqlFactory.getLogStoreSql(dbType).getAllLockSQL(lockTable, param.getTableName());
```

Comment:
param=null ？

Reply:
the param will not null,but the param's value maybe null, I had added the notice